docs/staking.md

```diff
@@ -0,0 +1,48 @@
+
+## Boosting Your Rewards {#boosting-your-rewards}
+
+Stakers can increase the pace at which they can claim their rewards by **collecting action points** and **boosting their rewards**. Action points can be acquired by performing certain actions that are incentivized by the Protocol. Initially, the only way to collect action points is to **participate in the governance of HydraDX** by [voting on community referenda](https://hydradx.subsquare.io/democracy/referenda) **using the staked HDX**.
+
+<div style={{textAlign: 'center'}}>
+  <img alt="login" src={useBaseUrl('/staking/rewards_bonding_curve.jpg')} width="500px" />
+</div>
+
+There are 2 factors which determine the amount of action points that stakers will receive: The **size of the vote** (relative to the total size of their staked HDX), and the **conviction multiplier**. The higher the conviction multiplier of the vote, the greater its weight. Keep in mind that voting with a conviction multiplier places a **temporary lock on the tokens**. Stakers looking to achieve the highest rewards boost would be voting with 6x conviction multiplier, thereby locking their HDX for 192 days. Just a reminder that **this lock is not related to staking** as such - instead, it is a standard feature of governance in the Polkadot ecosystem (more info [in our docs](/democracy_referenda#referenda-votes-weighing)).
+
+| Conviction Multiplier | Days Locked |
+|:---------------------:|:-----------:|
+| 0.1x                  |  0d         |
+| 1x                    |  6d         |
+| 2x                    |  12d        |
+| 3x                    |  24d        |
+| 4x                    |  48d        |
+| 5x                    |  96d        |
+| 6x                    |  192d       |
+
+## Claiming Your Rewards
+
+As they keep their HDX staked, **users accumulate rewards over time**. These rewards become claimable subject to a bonding curve which is influenced by the boosts from action points (see above).
+
+At any given time, **stakers can claim** (a portion of) their claimable rewards. By doing so, however, they **forfeit the remainder of their non-claimable rewards**. These rewards are **automatically transferred back to the staking rewards pot** which **redistributes them to all other stakers**. Furthermore, claiming **resets the past action points of the user**, sending users back to the beginning of the bonding curve for future rewards from staking.
```

Comment:
> ...**automatically transferred back to the staking rewards pot** which **redistributes them to all other stakers**.
I would remove "other" and leave just "...to all the skakers".
Rewards are always distributed to all the stakers. If user stay in the staking after claiming he will receive portion of his lost rewards back